homeassistant/components/hassio/ingress.py

```diff
@@ -183,7 +183,9 @@ def _init_header(request: web.Request, token: str) -> CIMultiDict | dict[str, st
         headers[name] = value
 
     # Inject token / cleanup later on Supervisor
-    headers[X_HASSIO] = os.environ.get("SUPERVISOR_TOKEN", "")
+    headers[
+        aiohttp.hdrs.AUTHORIZATION
```

Comment:
I guess that needs use the X- header

Reply:
Why? it's deprecated since 2020

Reply:
Not the old one, the new one. That is that we not overwrite headers that is used by the app

Reply:
Where is that announced? used? 
This request goes to the supervisor which handles it to the addon?